docs/sgx-binary-demo.md

```diff
@@ -0,0 +1,34 @@
+## Intel SGX 2 Static Binary Demo
+
+This demo is designed to illustrate a static binary running in a SGX Enclave.
+This demo *does not yet* include attestation.
+
+### Hardware Requirements
+SGX2 support is required to run this demo. Check with your hardware manufacturer
+to determine if your hardware has SGX2 support enabled. Alternatively, you may
+reference the following page for an incomplete but growing list of SGX enabled
+hardware: https://github.com/ayeks/SGX-hardware/#hardware-with-sgx2-support
+
```

Comment:
I still agree that we should refer to `sgx-show` here.

Reply:
Could you maybe provide an example of how to use it?

Reply:
Ah, @connorkuehl gave some pointers here: https://github.com/enarx/enarx/pull/582#discussion_r437073057